Project.toml

```diff
@@ -8,7 +8,7 @@ MLStyle = "d8e11817-5142-5d16-987a-aa16d5891078"
 NameResolution = "71a1bf82-56d0-4bbc-8a3c-48b961074391"
 
 [compat]
-MLStyle = "~0.3.1, ~0.4.1"
+MLStyle = "~0.4.1"
```

Comment:
I hope to have "~0.4.2" here because there is only a bugfix and a tiny improve([thautwarm/MLStyle#93](https://github.com/thautwarm/MLStyle.jl/issues/93), [thautwarm/MLStyle#98](https://github.com/thautwarm/MLStyle.jl/issues/98)) from 0.4.1 to 0.4.2.

Besides, please bump up the version so that I can call registrator.

Reply:
I don't think I understand. Would you like this to be `"~0.4.2"`? Maybe related to my lack of understanding here but what is the intension with the tilde? I don't think it makes a difference relative to just `"0.4.1"`, see https://julialang.github.io/Pkg.jl/v1/compatibility/#Tilde-specifiers-1.

Reply:
I've updated the bounds. Please let me know if it wasn't what you intended.